### DISCUSSION TITLE
Set fixed size for overlay image

### DISCUSSION THREAD

**Discussion by plugjoff** [claviska/SimpleImage#329] [Q&A]
I use this plugin to add a watermark to images with a lot of different sizes.
The watermark gets a different size for each image... How can I ensure that the watermark is always the same size? Independent of the size of the source file?

**Comment by claviska** [comment]
If you apply the overlay _after_ all resize operations, the dimensions won't change. Generally speaking, an overlay is the last effect applied to an image. If this isn't the behavior you're seeing, I suspect it would be a bug.

**Comment by plugjoff** [comment]
The problem is that we don't resize the images themselves. The page has a Pinterest like look that consists of images with all different sizes. The watermark should get the same size on every image.
